.circleci/config.yml

```diff
@@ -1,211 +1,180 @@
 version: 2.1
 jobs:
   tarball:
-    docker:
-      - image: circleci/buildpack-deps:stretch
+    machine:
+      image: ubuntu-1604:202007-01
+#    resource_class: large
```

Comment:
You have several of these commented out keys. Do you want to remove them all before merging?

Reply:
They are features I would like to use, but cost money.
It took me time to find the settings, so I wanted to keep them in case we ever pay circle CI.
I should probably add a comment to that effect.

Reply:
The only reason I didn't wholesale switch to github actions: at least circle CI *can* support more RAM and cores. It seems the only way to get more with github actions is to host the runners yourself!